backend/package.json

```diff
@@ -41,16 +41,20 @@
   "devDependencies": {
     "@babel/core": ">=7.0.0 <8.0.0",
     "@chax-at/transactional-prisma-testing": "^0.5.0",
+    "@faker-js/faker": "^7.5.0",
     "@prisma/generator-helper": "*",
     "@types/aws-lambda": "^8.10.70",
     "@types/memoizee": "^0.4.7",
+    "@types/uuid": "^8.3.4",
     "aws-sdk-client-mock": "^1.0.0",
     "builtin-modules": "3.2.0",
     "dotenv": "^16.0.0",
     "dotenv-cli": "^5.1.0",
+    "fishery": "^2.2.2",
     "graphql": "^16.5.0",
     "graphql-tag": "^2.12.6",
-    "prisma": "4.2.1"
+    "prisma": "4.2.1",
+    "uuid": "^9.0.0"
```

Comment:
not sure if `uuid` should be a dev dependency i think it will be used in production code at some point